libs/blocks/comparison-table/comparison-table.css

```diff
@@ -0,0 +1,509 @@
+  height: 12px;
+  width: 12px;
+}
+
+.comparison-table .header-content {
+  padding-bottom: var(--spacing-s);
+}
+
+.comparison-table .header-content.sticky {
+  position: sticky;
+  top: 0;
+  z-index: 9;
+  background-color: var(--color-gray-100);
+  width: 100%;
+  box-shadow: 0 1px 6px 0 rgb(0, 0, 0, 12%);
+}
+
+.comparison-table .header-content.sticky .header-content-wrapper {
+  justify-content: space-between;
+}
+
+.comparison-table .header-content-wrapper {
+  max-width: calc(100% - 60px);
+  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+  gap: var(--comparison-table-spacing);
+}
+
+.comparison-table .header-item {
+  display: flex;
+  flex-direction: column;  
+  gap: 4px;
+}
+
+.comparison-table .header-item:first-of-type {
+  margin-inline-end: var(--comparison-table-spacing);
+}
+
+.comparison-table .header-content-wrapper .header-item:first-child {
+  display: none;
+}
+
+.comparison-table .sub-header-item-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: var(--comparison-border-radius);
+  border: 1px solid var(--color-gray-300);
+  padding: var(--spacing-xxs);
+  position: relative;
+  justify-content: center;
+}
+
+.comparison-table .header-content.sticky .sub-header-item-container {
+  border: none;
+}
+
+.comparison-table .header-content.sticky .sub-header-item-container:nth-of-type(2),
+.comparison-table .header-content.sticky .sub-header-item-container:last-child .description,
+.comparison-table .header-content.sticky .sub-header-item-container .mobile-filter-select { 
+  display: none;
+}
+
+.comparison-table .sub-header-item-container .description {
+  text-align: center;
+  padding: var(--spacing-xxs);
+  font-size: var(--type-body-xxs-size);
+  line-height: 125%;
+  min-height: 15px;
+  display: block;
+}
+
+.comparison-table .sub-header-item-container .description.zero-height {
+  padding: 0;
+}
+
+.comparison-table .sub-header-item-container .btn-container {
+  display: flex;
+  justify-content: center;
+  gap: var(--spacing-xxs);
+  width: 100%;
+  margin-top: var(--spacing-xxs);
+  flex-wrap: wrap;
+}
+
+.comparison-table .action-area,
+.comparison-table .action-area :is(.con-button, strong, a) {
+  display: flex;
+}
+
+.comparison-table .action-area .con-button {
+  justify-content: center;
+}
+
+.comparison-table .sub-header-item-container:last-of-type {
+  border: none;
+  padding: 0;
+}
+
+.comparison-table .sub-header-item-container p:not(:has(picture, a)) {
+  display: flex;
+  align-items: center;
+  word-break: break-word;
+}
+
+.comparison-table .sub-header-item-container :is(p, h1, h2, h3, h4, h5, h6) {
+  text-align: center;
+  word-break: break-word;
+  color: var(--color-black);
+}
+
+.comparison-table .sub-header-item-container :is(h1, h2, h3, h4, h5, h6) {
+  font-size: var(--type-body-s-size);
+  font-style: normal;
+  font-weight: var(--type-detail-all-weight);
+  line-height: var(--type-body-s-lh);
```

Comment:
This looks a bit off - body and detail variables are used for headings. Were these the suggested values in Figma?